app/models/concerns/hud_reports/clients.rb

```diff
@@ -32,24 +32,45 @@ module HudReports::Clients
       }
     end
 
-    # Calculate the head of household’s number of years in the project. This can be done using the same
-    # algorithm as for calculating a client’s age as of a certain date. Use the client’s [project start date] and the
-    # [report end date] as the two dates of comparison. This calculation is time-based, not service-based, so a
-    # client with a [project start date] that is active for a year or longer in a night-by-night emergency shelter
-    # would require an Annual Assessment, regardless of how many bed night dates were recorded. It is
-    # important to use the “age” method of determining client anniversaries due to leap years; using “one year =
-    # 365 days” will eventually incorrectly offset the calculated anniversaries of long-term stayers.
-    private def annual_assessment_expected?(enrollment:, report_end_date: Date.current)
-      return false unless enrollment.present? && enrollment.head_of_household?
-
-      start_for_annual = enrollment.entry_date
-      end_date = [enrollment.last_date_in_program, report_end_date].compact.min
+    private def annual_assessment_expected?(hoh_enrollment:, enrollment:, report_end_date: Date.current)
```

Comment:
Should there be any check for age of client here?  Are annual assessments required on minors in an adult and child household?

Reply:
annual assessments are collected on all clients, but some questions may not be required for some populations.